activestorage/test/models/missing_table_test.rb

```diff
@@ -0,0 +1,22 @@
+# frozen_string_literal: true
+
+require "test_helper"
+require "database/setup"
+
+class ActiveStorage::MissingTableTest < ActiveSupport::TestCase
+  setup do
+    ActiveRecord::Migrator.down File.expand_path("../../db/migrate", __dir__)
+    ActiveRecord::Base.clear_cache!
+  end
+
+  teardown do
+    ActiveRecord::Migrator.migrate File.expand_path("../../db/migrate", __dir__)
+    ActiveRecord::Base.clear_cache!
+  end
+
+  test "creating a blog when Active Storage tables have not been setup" do
+    assert_raise ActiveStorage::Blob::MissingTableError do
+      create_blob data: "Hello world!"
+    end
+  end
```

Comment:
This doesn’t seem to warrant a new test case. I’d add it to `ActiveStorage::BlobTest`.